packages/evm/src/precompiles/14-kzg-point-evaluation.ts

```diff
@@ -29,6 +30,17 @@ export async function precompile14(opts: PrecompileInput): Promise<ExecResult> {
     )
   }
 
+  if (opts.gasLimit < gasUsed) {
+    if (opts._debug !== undefined) {
+      opts._debug(`KZG_POINT_EVALUATION (0x14) failed: OOG`)
+    }
+    return OOGResult(opts.gasLimit)
```

Comment:
Can you add a quick test for this? Should help with coverage and should be easy enough to add? Otherwise, I can do it later today